openadmet/models/tests/test_data/cyp3a4_anvil_lgbm_model_dir/anvil_recipe.yaml

```diff
@@ -0,0 +1,80 @@
+data:
+  anvil_dir: file:///store/projects/OpenADMET_WORK/initial_runs/CYP3A4_cross_val
+  cat_entry: null
+  input_col: OPENADMET_SMILES
+  resource: cyp3a4_chembl_permissive.csv
+  target_col: pchembl_value_mean
```

Comment:
Pass target column through to eval, such that labelling is possible and convenient.